docs/man_pages/project/configuration/resource-create.md

```diff
@@ -0,0 +1,45 @@
+resource create
+==========
+
+Usage | Synopsis
+------|-------
+General | `$ appbuilder resource create [--icon <File Path>] [--splash <File Path>] [--force]`
+
+Creates image resources for all mobile platforms from a single high-resolution image and saves them to the App_Resources folder of the current project. The image source must be a PNG image. <% if(isHtml) { %>If you do not provide any option, the Telerik AppBuilder CLI shows an interactive prompt to let you set the remaining details. <% } %>
+
+<% if(isConsole) { %>
+<% if(isMobileWebsite)  { %>
+WARNING: This command and its extended commands are not applicable to mobile website projects. To view the complete help for this command, run `$ appbuilder help resource create`
+<% } %>
+<% } %>
+<% if((isConsole && (isCordova || isNativeScript)) || isHtml) { %>
+
+### Options
+* `--icon` - If set, will create icons based on a single high-resolution image.
```

Comment:
Creates all required icons for all mobile platforms from a single high-resolution image. The source image must be at least a XxY pixels `PNG` image.